TAF/testCaseModules/keywords/core-data/coreDataAPI.robot

```diff
@@ -261,6 +261,7 @@ Generate event sample
         Append to List  ${readings}  ${reading}
     END
     ${event}=  Load data file "core-data/event_data.json" and get variable "${event_data}"
+    Set to dictionary  ${event}         apiVersion=${api_version}
```

Comment:
Doesn't need to set `apiVersion` here because it has been set in event test data.